tuxemon/npc.py

```diff
@@ -886,6 +886,9 @@ def find_tech(self, tech: str) -> Optional[Technique]:
 
         return None
 
+    def give_money(self, amount: int) -> None:
+        self.money["player"] = self.money.get("player") + amount
```

Comment:
Will player always be here?

`+= amount` if it is, otherwise do `.get("player", 0) + amount`

Reply:
Ok, thanks